packages/workflow/src/Expression.ts

```diff
@@ -513,6 +513,10 @@ export class Expression {
 		const returnData: INodeParameters = {};
 		// eslint-disable-next-line no-restricted-syntax
 		for (const [key, value] of Object.entries(parameterValue)) {
+			// TODO: Remove these. This is a temp hack to get it working with the
+			// resource locator component.
+			// @ts-ignore
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
 			returnData[key] = resolveParameterValue(value, parameterValue);
```

Comment:
should this be removed before merging?

Reply:
I've changed this to do some type casts. It's happening because the resource locator value is the only properly typed object that gets to that point. This causes some oddities with the types. I just cast them because it shouldn't cause any issues and it produces the expected results.